addons/rose/.template-lintrc.js

```diff
@@ -31,5 +31,11 @@ module.exports = {
         'no-passed-in-event-handlers': false,
       },
     },
+    {
+      files: '**/tests/**/*.{js,gts,gjs}',
```

Comment:
None blocking question: this line, was add after running the command to update ember or you add it manually?

Reply:
@calcaide I added manually.  I think after the discussion with you and the team about formatting in our test files, I reached out to the cloud team and this is what they currently do.  I mean we probably don't need the `.gts` format, but I hope that we will get to `.gjs` someday